### ISSUE TITLE
FFT values are alway minus infinity (in the console.log). 

### ISSUE THREAD

**Issue by cordial** [Tonejs/Tone.js#931]
**Describe the bug**

This is carrying on from the issues I still have here - 
https://github.com/Tonejs/Tone.js/issues/897

**To Reproduce**

```import React from "react";
import Sketch from "react-p5";
import p5Types from "p5"; //Import this for typechecking and intellisense
//import "p5/lib/addons/p5.sound";
import { FFT } from "tone";
interface ComponentProps {
	//Your component props
}

const NFT = () => {
	let x = 50;
	const y = 50;
	let fft = new FFT(32).toDestination();

	//See annotations in JS for more information
	const setup = (p5: p5Types, canvasParentRef: Element) => {
		p5.createCanvas(500, 500).parent(canvasParentRef);
		//amp = new p5.constructor.Amplitude();
	};

	const draw = (p5: p5Types) => {
		p5.background(fft.getValue()[0]);
		p5.ellipse(x, y, 70, 70);
		console.log(fft.getValue());
		x++;
	};

	return <Sketch setup={setup} draw={draw} />;
};
export default NFT;
```

**Expected behavior**
I want to get values over time that mean I can change the background colour to the pulse of the sine wave oscillators that are playing (they are playing in other classes but assume I have a metronome and an oscillator playing at frequency X). 
**What I've tried**

I've tried to connect to the destination, to the metronome player and to the oscillator
I've tried to output all the params of FFT and also looked at Analyser and Waveform. All I want to do is change colour of background to match the noise being played. 

**Additional context**
This is a react next.js project...

**Comment by tambien** [comment]
Make sure you have something connected to the input of FFT, otherwise it will just be reading silence which is `-Infinity dB`

**Comment by cordial** [comment]
thanks,  I managed to get an output from Waveform using an oscillator. But the numbers seem much smaller than I would expect, I've attached the output of a 5hz Oscillator, this is the 1st element of each 1024 array being outputted by the draw function i.e. every animation frame above - 

<img width="476" alt="Screenshot 2021-08-16 at 20 04 04" src="https://user-images.githubusercontent.com/419893/129616440-afcbd331-c251-45aa-a796-b33ee65f3dbb.png">

I would expect the numbers to get much nearer to 1 and minus 1, but they're not moving from 0 very much at all. Is the output tied to the volume I am playing the oscillator at, or some other param?
